adapter/config/default_config.go

```diff
@@ -117,9 +119,9 @@ var defaultConfig = &Config{
 				HealthyThreshold:   2,
 			},
 			Retry: upstreamRetry{
-				MaxRetryCount:        5,
-				BaseIntervalInMillis: 25,
-				StatusCodes:          []uint32{504},
+				MaxRetryCount: 5,
+				BaseInterval:  25 * time.Millisecond,
```

Comment:
Are we retrying on millisecond intervals?
25
50
100
200
400
?
Should we do something like this instead? and stop at 4th retry (total requests = 5)

500
1000
2000
4000

Reply:
25ms the default behavior. So I think it is better to go with shorter retry intervals.